tests/integration/test_kalman_filter_with_sparse_sites.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import tensorflow as tf
+import pytest
+from gpflow.config import default_float
+
+from markovflow.kernels.matern import Matern12
+from markovflow.mean_function import LinearMeanFunction
+from markovflow.models.gaussian_process_regression import GaussianProcessRegression
+from markovflow.kalman_filter import KalmanFilterWithSparseSites, UnivariateGaussianSitesNat
+from markovflow.likelihoods import MultivariateGaussian
+
+
+@pytest.fixture(name="kalman_gpr_setup")
+def _setup(batch_shape):
+    """
+    FIXME: Currently batch_shape isn't used.
+    """
+    time_grid = np.arange(0.0, 1.0, 0.01)
+    time_points = time_grid[::10]
+    observations = np.sin(12 * time_points[..., None]) + np.random.randn(len(time_points), 1) * 0.1
+
+    input_data = (
+        tf.constant(time_points, dtype=default_float()),
+        tf.constant(observations, dtype=default_float()),
+    )
+
+    observation_covariance = 1.0  # Same as GPFlow default
+    kernel = Matern12(lengthscale=1.0, variance=1.0, output_dim=observations.shape[-1])
```

Comment:
could you try to have some state offset into the kernel?
That will test whether the Kf class deals well with offsets in the ssm (I m not 100% sure it will)
and we need this with our linearized SDEs

Reply:
It does! I added it to the test script.